src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -215,13 +274,14 @@ function PaymentMethodList(props) {
                     iconHeight={item.iconSize}
                     iconWidth={item.iconSize}
                     badgeText={shouldShowDefaultBadge(filteredPaymentMethods, item.isDefault) ? translate('paymentMethodList.defaultPaymentMethod') : null}
-                    wrapperStyle={item.wrapperStyle}
+                    wrapperStyle={[styles.cardMenuItem]}
```

Comment:
This supports both objects and array of objects. Since it's only one style, I would leave as an object (like before):

```suggestion
                    wrapperStyle={styles.cardMenuItem}
```

Reply:
I think this is outdated 😄